sql/education_teachers(grade1-12).sql

```diff
@@ -0,0 +1,270 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+SET search_path = public, pg_catalog;
+
+
+ALTER TABLE IF EXISTS ONLY public.education_teachers DROP CONSTRAINT IF EXISTS education_teachers_pkey;
+DROP TABLE IF EXISTS public.education_teachers;
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: education_teachers; Type: TABLE; Schema: public; Tablespace:
+--
+
+CREATE TABLE education_teachers(
+    geo_level character varying(15) NOT NULL,
+    total integer NOT NULL,
+    "teacher type" character varying(128) NOT NULL,
+    geo_code character varying(10) NOT NULL
+);
+
+
+--
+-- Data for Name: education_teachers; Type: TABLE DATA; Schema: public
+--
+
+
+
+COPY education_teachers( geo_level, total, "teacher type", geo_code) FROM stdin WITH DELIMITER ',';
+country,109109,Approved teachers,NP
+country,38420,Rahat Teachers,NP
+country,147588,Total Teachers,NP

```

Comment:
@pratimakandel 

Are there teachers who are neither "approved" nor "rahat"? 

We don't need the "total" row because the framework will add together all of the other categories to get a total. If there are teachers who are neither "approved" nor "rahat", then we should have an "other" category instead of a "total" category. See, for example, some of the other SQL files that have categories but no rows for "total", like
https://github.com/Code4Nepal/nepalmap_app/blob/dev/sql/age.sql
https://github.com/Code4Nepal/nepalmap_app/blob/dev/sql/drinkingwatersource.sql

The reason I asked about "other" is that the national "total" is greater than the sum of "approved" and "rahat". In a quick glance through the district-level data, I see "total" usually matches the sum of "approved" and "rahat". I wonder if there is a district that has an incorrect total in it.